src/routes/auth-old/+page.svelte

```
@@ -0,0 +1,538 @@
+<script lang="ts">
+	import { toast } from 'svelte-sonner';
+
+	import { onMount, getContext, tick } from 'svelte';
+	import { goto } from '$app/navigation';
+	import { page } from '$app/stores';
+
+	import { getBackendConfig } from '$lib/apis';
+	import { ldapUserSignIn, getSessionUser, userSignIn, userSignUp } from '$lib/apis/auths';
+	import { trackUserRegisteredWithAuth0, trackUserLoggedInWithAuth0 } from '$lib/services/analytics';
+
+	import { WEBUI_API_BASE_URL, WEBUI_BASE_URL } from '$lib/constants';
```

ESLint

[error] 'WEBUI_API_BASE_URL' is defined but never used.

(@typescript-eslint/no-unused-vars)